circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  node:
+    version: 6.9.2
```

Comment:
that's on the dependencies section, not sure is the actual node version here is still needed , but circle don't have a simple multiple node setup.